frontend/src/components/input/TextEditor.vue

```diff
@@ -318,12 +318,26 @@ watch(
 
 const showButtons = computed(() => props.editable && hasFocus.value)
 
+watch(
+  () => hasFocus.value,
+  () => {
+    if (props.editable) {
+      // When the TextEditor is editable and has focus, the invisibleCharacters should be displayed
+      commands.toggleActiveState()(editor.state, editor.view.dispatch)
+    }
+  },
+)
+
```

Comment:
I would remove the commands.setActiveState from the onMounted method and put it all in one place:

```suggestion
watch(
  () => hasFocus.value,
  () => {
    if (props.editable) {
      // When the TextEditor is editable and has focus, the invisibleCharacters should be displayed
      commands.setActiveState(props.editable && hasFocus.value)(
        editor.state,
        editor.view.dispatch,
      )
    }
  },
  { immediate: true },
)

```

Reply:
Thanks for the hint 🙏 . I changed it and love the simplified version. However, I added it without the if-statement as it is not needed.